src/extension/serializer.ts

```diff
@@ -886,9 +930,9 @@ export class GrpcSerializer extends SerializerBase {
       typeof metadata['runme.dev/frontmatter'] !== 'string'
     ) {
       log.warn('no frontmatter found in metadata')
-      return {
+      return new Frontmatter({
         category: '',
-        tag: '',
+        // tag: '',  // buf-es does not have tag property, but the old timostam/prototype-ts did
```

Comment:
I think you may need to update the dependency for the stateful es library.
It looks like tag was added in September
https://github.com/stateful/runme/commit/77ace522e46b2fb111e9800bf5adb8369f7f648b

I don't see a direct dependency on that library in package.json
https://github.com/hotpocket/vscode-runme/blob/f9f3e1bb4be8c146bf8b15e20afd4667943b7e05/package.json#L1365

So I wonder if its getting pulled in indirectly via jlewi. 

At any rate try running

```
npm install @buf/stateful_runme.bufbuild_es@1.10.0-20241021161352-11c142f88aee.1
```

To update the dependency and see if that version has the tag field.

Reply:
Done.
https://buf.build/jlewi/foyle/commits

Per https://buf.build/jlewi/foyle/deps seems to be depending on latest version of stateful now.

Reply:
this seems to resolve the issue. Thanks!